### ISSUE TITLE
TemplateSyntaxError at /wearpants/

### ISSUE THREAD

**Issue by glsdesign** [dylanleigh/PriceAndNutritionTrackingSystem#3]
Hi Dylan, your application really intrigues me, it can be the most complete I've ever seen. I followed your instructions but trying to access http://127.0.0.1:8000 it gives the following error in the rendered webpage:

```
TemplateSyntaxError at /wearpants/

'staticfiles' is not a registered tag library. Must be one of:
admin_list
admin_modify
admin_urls
cache
css_visuals
debugger_tags
highlighting
i18n
indent_text
l10n
log
static
syntax_color
truncate_letters
tz
widont
```

Any hint?

**Comment by dylanleigh** [comment]
Looks like "staticfiles" was deprecated and removed between Django 2.x and 3.x - I'll update to static

**Comment by dylanleigh** [comment]
Thanks for reporting this - I've updated the static library and bumped the minimum version to the latest LTS release of Django to catch any other deprecated things.

Fixed in commit 2c24966b78bf872c1e0c02e0e1b78274e1705fc0
